conjurapi/policy.go

```diff
@@ -29,5 +17,6 @@ func (c *Client) LoadPolicy(policyId string, policy io.Reader) (map[string]inter
 		return nil, err
 	}
 
-	return wrapper.LoadPolicyResponse(resp)
+	obj := make(map[string]interface{})
```

Comment:
what is `obj` here, can we use a more descriptive name? this method could probably also use a docstring to explain what is passed in and returned.